src/docs/releases/2.1.0.md

```diff
@@ -50,6 +50,15 @@ The following properties of `RegistrationSettings` are now deprecated and will b
 
 Previously, the `UsersCanRegister` property controlled which types of registration were allowed. With this update, this property is obsolete and will be removed in a future release. To enable site registration now, simply activate the **User Registration** feature.
 
+### Roles Feature
```

Comment:
```suggestion
### Roles Assignment Permissions Consolidated
```

Reply:
Typically we use the feature or module name in the header. This was we add all items related to it.

Reply:
We have all the power to change that! :) This change affects how you use both the Roles and the Users features

Reply:
Yeah but I like keeping the title as the feature :). Do I have the power not to change that too? lol

Reply:
We should rather have titles, all of them, that convey what the changes are. And correspondingly, not group multiple unrelated changes together under the same title even if they affect the same module/feature. People will scan the titles and it's more useful to get an excerpt on what changed rather than where changes were made. I.e.:

- External Authentication Logic Factored out from the Users Feature
- User Registration Feature Not Needed for External Authentication
- Roles Assignment Permissions Consolidated
- New Azure Communication Services SMS Feature (this is already like that)
- Azure Communication Services Email Feature Name Update

Reply:
Because you know where the code for everything is. Except for a select few, this is not the case.

Reply:
not really. Everything is feature based. so if someone looking for changes on a feature will find them in one place.

Reply:
Do you expect every site maintainer to always know which functionality comes from which feature, especially on the code level? Why is the title of this PR not just "Roles Feature" too?

All I'm advocating here, is that instead of having a mechanical list of feature names so everyone has to read pretty much everything (or just skip them and not get informed), have titles that provide some information about the change apart from just the feature name, i.e. serve as good titles. I provided examples of this for each existing title above. And then again, you've also done this with "New Azure Communication Services SMS Feature" and actually with "Azure Communication Services Email Feature Update" too (but the latter can be better, I think, see "Azure Communication Services Email Feature **Name** Update"). I really fail to see what's there to debate about.

That we had only feature names listed before is not a good argument.

Reply:
> Do you expect every site maintainer to always know which functionality comes from which feature, especially on the code level? Why is the title of this PR not just "Roles Feature" too?

No by I expect them to know what feature they are using or have enabled. So if there is any change impacting that feature, they can see it listed and act on it "if needed".

At least this make sense to me. If you want to change that, I would suggest submitting a PR updating the release notes.

Reply:
OK, I'll open a PR.